owncloudApp/src/main/java/com/owncloud/android/presentation/security/passcode/PassCodeActivity.kt

```diff
@@ -434,15 +444,33 @@ class PassCodeActivity : AppCompatActivity(), NumberKeyboardListener, EnableBiom
             ACTION_REMOVE -> {
                 return PasscodeAction.REMOVE
             }
+
             ACTION_CREATE -> {
                 return PasscodeAction.CREATE
             }
+
             else -> {
                 return PasscodeAction.CHECK
             }
         }
     }
 
+    override fun onKeyUp(keyCode: Int, event: KeyEvent?): Boolean {
+        return when (keyCode) {
+            KeyEvent.KEYCODE_TAB -> {
+                findViewById<View>(R.id.lock_time).requestFocus()
+                true
+            }
```

Comment:
Why don't you add here a blank line in between the `when` options? in the code above 👆  , you added blanks in every line of existing code

Reply:
Changed